### PR TITLE
Add `_dashprivate_onFigureModified`

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**

### CODE REVIEW

File: src/fragments/Graph.react.js [hunk 1]
```diff
@@ -352,6 +353,14 @@ class PlotlyGraph extends Component {
         if (this.props.extendData) {
             this.mergeTraces(this.props, 'extendData', 'extendTraces');
         }
+
+        if (
+            (this.props.prependData?.length ?? 0) +
+                (this.props.extendData?.length ?? 0) >
+            0
```
Comment (alexcjohnson) [inline src/fragments/Graph.react.js:360]: Just curious, is there a reason you prefer this to
`if (this.props.prependData?.length || this.props.extendData?.length)`
?

**Review by alexcjohnson — APPROVED** [review]
💃

File: src/fragments/Graph.react.js (hunk 1 above)
Comment (Marc-Andre-Rivet) [reply src/fragments/Graph.react.js:360]: None whatsoever, will update.

Comment (Marc-Andre-Rivet) [reply src/fragments/Graph.react.js]: https://github.com/plotly/dash-core-components/pull/865/commits/a156dccb05f916968d0c2170d9a7a86a9b8acb8a

**Pull request merged**